bitwardenrs/values.yaml

```diff
@@ -138,6 +138,15 @@ persistence:
   ## Use existing Persistent Volume Claim
   # existingClaim:
 
+
+# define custom "Volumes" config instead of PVC 
+customVolumes:
+  enabled: false
+  config: {}
```

Comment:
I think this could be a list in order to allow multiple mounts, so
```
config: []
 # - name: example
 #   hostPath:
 #     path: "/examplefolder/bitwardenrs"
```

Reply:
If you add multiple mounts, you will also need to customize the Pod volume mount.

Reply:
True, it was mostly just a custom mountpoint for all the data, not multiple custom-random mountpoints.

And I agree what @sgissi says: the usecase is quite limited and reworking it is a more significant rework.